src/app/app.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+
+@Pipe({
```

Comment:
So yeah, you could include this in a core module but it's probably makes more sense to included it in a "Shared Module". The shared module being a module that is common material used across your other modules. Believe it or not, a shared service probably shouldn't go in a shared module because lazy loaded modules will get their own instances. Services should go in a core module in that they're added to that core module's injector. 

Does that make sense? I'm seeking a sanity check.